Copying Github templates from T4Rec (#165)
NVIDIA-Merlin/models: .github/ISSUE_TEMPLATE/bug-report.md

---
name: "\U0001F41B Bug Report"

> about: Submit a bug report to help us improve Transformers4Rec

Reviewer: ```suggestion
about: Submit a bug report to help us improve Merlin Models
```

title: "[BUG]"
labels: "status/needs-triage, bug"
assignees: ''

---

### Bug description
<!-- A clear and concise description of what the bug is. -->

### Steps/Code to reproduce bug
<!-- Follow this guide http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly. -->
1. 
2. 
3. 

### Expected behavior
<!-- A clear and concise description of what you expected to happen. -->

### Environment details

> - Transformers4Rec version: 

Reviewer: ```suggestion
- Merlin version: 
```

- Platform:
- Python version:

> - Huggingface Transformers version:

Reviewer: ```suggestion
```
